src/widgets/Catalog/filters/field.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import ArrowDown from './asserts/arrow-down.svg';
+import ArrowUp from './asserts/arrow-up.svg';
+
+type Props = {
+  id: string;
+  name: string;
+};
+
+const FiltersField = (props: Props) => {
```

Comment:
```suggestion
const FiltersField: FC<Props> = ({ name }) => {
```
В целом разницы нет, но у нас в остальных компонентах используется FC для типизации пропсов и деструктуризация тут же в скобочках. Думаю будет хорошо соблюдать единый стиль.